account_consolidation/models/account_move.py

```diff
@@ -14,13 +14,18 @@ class AccountMove(models.Model):
         string='Consolidated from Company',
         readonly=True
     )
+    to_be_reversed = fields.Boolean(
+        string="To Be Reversed",
+        copy=False,
+        help="Check this box if your entry has to be reversed at the end "
```

Comment:
This field definition looks like a copy-paste from https://github.com/OCA/account-financial-tools/blob/11.0/account_reversal/models/account_move.py#L15
If it's now only used to find consolidation entries to be reversed, I'd remove string and help, and add a comment like "Technical field to search consolidation entries that must be reverted on the next run (previously defined in account_reversal)" and set it readonly.